src/Service.Tests/Configuration/ConfigurationTests.cs

```diff
@@ -2176,7 +2178,7 @@ private static JsonContent GetJsonContentForCosmosConfigRequest(string endpoint,
                     // exception. To prevent this, CosmosClientProvider parses the token and retrieves the "exp" property
                     // from the token, if it's not valid, then we will throw an exception from our code before it
                     // initiating a client. Uses a valid fake JWT access token for testing purposes.
-                    RuntimeConfig overrides = new(null, new DataSource(DatabaseType.CosmosDB_NoSQL, "AccountEndpoint=https://localhost:8081/;", new()), null, null);
+                    RuntimeConfig overrides = new(null, new DataSource(DatabaseType.CosmosDB_NoSQL, "AccountEndpoint=https://localhost:8081/;", new()), null, new(new Dictionary<string, Entity>()));
```

Comment:
nit: it will be easier to understand if this argument names are provided here.

Reply:
fixed.